LICENSE

```diff
@@ -0,0 +1,21 @@
+mwdb-core is distributed under the terms of the GNU Affero General Public License
+as published by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version, with a special exception described below.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; 
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+See the GNU Affero General Public License for more details.
+
+A copy of GNU Affero General Public License (AGPL) is included in this distribution,
+in the file LICENSE.AGPL. If you do not have the source code, it is available at: 
+	
+	https://www.gnu.org/licenses/agpl-3.0.html
+
+This file forms part of the mwdb-core software for which the following exception is added:
```

Comment:
"This file" refers to the `LICENCE` or `LICENSE.AGPL`?

Reply:
It refers to `LICENSE` file, describing exceptions that apply to mwdb-core. This file is a part of mwdb-core, but can be modified and reused for another project. That's what I've done with https://github.com/roundcube/roundcubemail/blob/master/README.md

I've modified it a bit for disambiguation:
```diff
- This file
+ LICENSE file
```